astropy/utils/iers/iers.py

```diff
@@ -399,7 +403,8 @@ class IERS_A(IERS):
     Notes
     -----
     The IERS A file is not part of astropy.  It can be downloaded from
-    ``iers.IERS_A_URL``.  See ``iers.__doc__`` for instructions on how to use
+    ``iers.IERS_A_URL`` or ``iers.IERS_A_URL_MIRROR``.
+    See ``iers.__doc__`` for instructions on how to use
     it in ``Time``, etc.
```

Comment:
Wrap to one line? If too long, "for instructions on use in ``Time``, etc."

Reply:
Sure. Looks like I broke a test in https://travis-ci.org/astropy/astropy/jobs/470659497 (which didn't happen locally) anyway, so I can fold this into the follow up commit.